packages/core/src/modules/credentials/formats/jsonld/JsonLdCredentialFormatService.ts

```diff
@@ -0,0 +1,316 @@
+import type {
+  FormatAcceptOfferOptions,
+  FormatAcceptProposalOptions,
+  FormatCreateOfferOptions,
+  FormatCreateOfferReturn,
+  FormatCreateProposalOptions,
+  FormatCreateReturn,
+  FormatProcessOptions,
+} from '..'
+import type { Attachment } from '../../../../decorators/attachment/Attachment'
+import type { SignCredentialOptionsRFC0593 } from '../../../vc/models/W3cCredentialServiceOptions'
+import type {
+  FormatAcceptRequestOptions,
+  FormatAutoRespondOfferOptions,
+  FormatAutoRespondProposalOptions,
+  FormatAutoRespondRequestOptions,
+  FormatCreateProposalReturn,
+  FormatCreateRequestOptions,
+} from '../CredentialFormatServiceOptions'
+import type { JsonLdCredentialFormat } from './JsonLdCredentialFormat'
+
+import { Lifecycle, scoped } from 'tsyringe'
+
+import { AriesFrameworkError } from '../../../../../src/error'
```

Comment:
```suggestion
import { AriesFrameworkError } from '../../../../error'
```
Goes one level too high, which is problematic once bundled :)